examples/text-generation/checkpoint_utils.py

```diff
@@ -66,13 +66,35 @@ def model_is_bloom(config):
     return "bloom" in config.architectures[0].lower()
 
 
+def get_optimized_model_name(config):
+    model_names = ["bloom", "gpt2", "opt", "gptj", "gptneox"]
+    for model_name in model_names:
+        if model_name in config.architectures[0].lower():
```

Comment:
```suggestion
    model_names = ["bloom", "gpt2", "opt", "gptj", "gpt_neox"]
    for model_name in model_names:
        if model_name == config.model_type:
```
My previous implementation was not very clean, this is better XD

Reply:
done